app/services/api/v1/find_or_upsert_group.rb

```diff
@@ -8,7 +8,7 @@ def reload
       private
 
       def lookup_attributes
-        [:name_en]
+        [:name_en, :name_cn]
```

Comment:
not sure this is right, and in the following `find_or_upsert` services. this is used to look up entities in a hash dictionary created like this:
`@dict = Hash[Group.all.map { |e| [e.name_en, e] }]`
the `_cn` attribute is not part of the hash key, so this will look up nothing.

Reply:
I know, this is actually an hack that fixes another issue. At https://github.com/Vizzuality/gefc-api/blob/b2749db56c7c6a2a1d82008686f23081b50bf8b2/app/services/api/v1/find_or_upsert_entity.rb#L44 we should return if the attributes_for_update values are the same as the ones already present on the object. Since we're not doing that validation, we basically keep updating the same objects over and over again, with the exact same name_cn. This hack takes care of that in a very dirty way, but this is rails and magical iteration and comparisons on ActiveRecord fields seems like a very small rabbit hole I would get lost in pretty quickly. Maybe you know how to navigate this properly?

Reply:
yes, I think you could try on the following lines to replace
`entity.update(attributes_for_update)`

with 
```
entity.assign_attributes(attributes_for_update)
entity.save
```

and I think this would not fire and update if attributes have not changed.